### PR TITLE
removing dependency to tensorflow

### PR DESCRIPTION
**Pull request by triwahyuu**
here I used the builtin [tensorboad writer from PyTorch](https://pytorch.org/docs/stable/tensorboard.html) which was officially supported in PyTorch 1.2.0. I also removed the warning for the use of torch.uint8 for indexing.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,7 @@
 numpy
-torch>=1.0
+torch>=1.2
```
Comment (Flova) [inline requirements.txt:2]: Is this bump really needed?

File: utils/utils.py [hunk 2]
```diff
@@ -266,7 +266,7 @@ def non_max_suppression(prediction, conf_thres=0.5, nms_thres=0.4):
 
 def build_targets(pred_boxes, pred_cls, target, anchors, ignore_thres):
 
-    ByteTensor = torch.cuda.ByteTensor if pred_boxes.is_cuda else torch.ByteTensor
+    BoolTensor = torch.cuda.BoolTensor if pred_boxes.is_cuda else torch.BoolTensor
```
Comment (Flova) [inline utils/utils.py:269]: This is not part of this pull request. Could you remove it. Otherwise, this PR seems fine.

**Review by Flova — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (triwahyuu) [reply requirements.txt:2]: as I mentioned in the description, tensorboard support in pytorch is started on version 1.2

File: utils/utils.py (hunk 2 above)
Comment (triwahyuu) [reply utils/utils.py:269]: sory that this might be unrelated, I'm trying to remove the warning from pytorch, as it is recommended to use BoolTensor for boolean operation, e.g. tensor masking

Comment (Flova) [reply utils/utils.py:269]: I know that issue and it is fixed on the current master due to PR #541

File: requirements.txt (hunk 1 above)
Comment (Flova) [reply requirements.txt:2]: Oh sorry nvm

File: utils/utils.py (hunk 2 above)
Comment (triwahyuu) [reply utils/utils.py:269]: ohh sorry, will remove this one

**Review by Flova — APPROVED** [review]
Thanks for your contribution!
